index.js

```diff
@@ -127,7 +127,7 @@ class LedgerBridgeKeyring extends EventEmitter {
           this.hdk.chainCode = Buffer.from(payload.chainCode, 'hex')
           resolve(payload.address)
         } else {
-          reject(payload.error || 'Unknown error')
+          reject(new Error(payload.error || 'Unknown error'))
```

Comment:
Nit: We could consider this instead:

```suggestion
          reject(payload.error || new Error('Unknown error'))
```

If we trust that `payload.error` is an error (which it probably will be once #103 is merged). Right now it would double-wrap the error. But that's fine - an Error is still a valid constructor parameter for an Error. It just means the log will start with `Error: Error:` instead of `Error:`.

Reply:
Should we do that with the rest of the file? I was following that pattern.

Reply:
Oh! Good point, I hadn't noticed this is how it was done elsewhere.

I guess we should change this everywhere, to avoid the duplicate `Error:` prefix, but it doesn't matter a great deal. We can merge this PR as-is if you want, since it does restore consistency.

Reply:
I'll make another PR for it.

Reply:
https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/105